.huskyrc.yml

```diff
@@ -0,0 +1,3 @@
+hooks:
```

Comment:
Any reason for the change to the .yml file instead of leaving it in package.json? It no longer matches the way we do husky precommits in the js-sdk with this change.

Reply:
New version of husky is forcing a new format that doesn't use the `scripts` field in `package.json`. When I updated I figure it made sense to move that out of `package.json` since the file was getting pretty huge.

Reply:
Gotcha, we'll need to make that change to js-sdk's husky config too